prover/src/constraints/evaluator.rs

```diff
@@ -314,11 +303,10 @@ impl<'a, A: Air, E: FieldElement<BaseField = A::BaseField>> ConstraintEvaluator<
 
         // merge transition constraint evaluations into a single value and return it;
         // we can do this here because all transition constraints have the same divisor.
-        self.transition_constraints.aux_constraints().iter().fold(E::ZERO, |result, group| {
-            let (power, offset_exp) = (group.degree_adjustment(), group.domain_offset_exp());
-            let xp = domain.get_ce_x_power_at(step, power, offset_exp);
-            result + group.merge_evaluations::<E::BaseField, E>(evaluations, xp)
-        })
+        let result = evaluations.iter().zip(self.transition_constraints.aux_constraint_coef().iter()).fold(E::ZERO, |acc, (&const_eval, &coef)| {
+            acc + coef * const_eval
+        });
+        result
```

Comment:
minor nit: similar comment re formatting and `result` variable as above.

Reply:
Done